### DISCUSSION TITLE
Signal API no longer working after restarting the Docker container

### DISCUSSION THREAD

**Discussion by Jones1066** [bbernhard/signal-cli-rest-api#478] [General]
I've had the Signal API Docker container working well on a Proxmox LXC.  The trouble started after I rebooted the Proxmox server and restarted the docker container for Signal.

Previously working API calls were returning "error": "Specified account does not exist" so I tried to reregister the signal account and then received "error":"Failed to register: StatusCode: 499 (DeprecatedVersionException)" 

Any ideas would be much appreciated.

Thanks,
Jeff

**Comment by RJ-Make** [comment]
I just ran into a "...is the number registered" error after a server reboot. I update to the latest version .81 just released and I'm back up and running.

**Reply by Jones1066** [reply]
Thanks for the reply. I've dropped and reloaded the docker container a number of times without any luck.

I've also tried reregistering the number but I think it's failing now due to the number already having been registered. The question I have is how do I get the number set up again with the Signal docker instance?

Pulling my hair out!

Thanks,
Jeff

**Comment by bbernhard** [comment]
Please try to also remove the signal-cli-config folder on the filesystem before you re-register your number.

**Reply by Jones1066** [reply]
I've gone back through from the start (after a night's sleep) and now things are sorted. I was running curl from the wrong host! 

Thanks for responding and I'm very sorry for the trouble.
